pravega/pravega
# Issue 6732: Unexpected Pravega pod restarted in system tests

Signed-off-by: Kuldeep Kumar <kuldeep.kumar3@dell.com>

**Change log description**  
There are system test failure due to `java.lang.AssertionError: Failing test due to Pravega pod restart`. This happens to different test cases at different executions.

**Purpose of the change**  
Fixes #6732 

**What the code does**  
Adding antiaffinity for segment store pod w.r.t zookeeper pod. 

**How to verify it**  
Running the system tests multiple time and verify that there are no failures due to `java.lang.AssertionError: Failing test due to Pravega pod restart` and also zookeeper pod should not restart.
